index.js

```diff
@@ -8,75 +12,24 @@ const addNew = document.querySelector('.Add-New');
 const Contact = document.querySelector('.Contact');
 const contactSection = document.querySelector('.contact');
 const dataContainerTitle = document.querySelector('.dataContainerTitle');
+const time = document.querySelector('.time');
 
-class BookShelv {
-  books = JSON.parse(localStorage.getItem('books')) || [];
-
-  addBooks() {
-    const addedBook = {
-      title: titleInput.value,
-      author: authorInput.value,
-    };
-    this.books.push(addedBook);
-
-    localStorage.setItem('books', JSON.stringify(this.books));
-  }
-
-  displayBooks() {
-    this.books.forEach((hold) => {
-      const elementContainer = document.createElement('div');
-      elementContainer.classList.add('element-container');
-      const infoContainer = document.createElement('p');
-      const removeButton = document.createElement('button');
-      removeButton.classList.add('removebtn');
-      removeButton.type = 'button';
-
-      infoContainer.innerText = `"${hold.title}" by ${hold.author}`;
-      removeButton.innerText = 'Remove';
-      elementContainer.append(infoContainer, removeButton);
-      dataContainer.appendChild(elementContainer);
-    });
-  }
+const bookshel = new BookShelv(dataContainer);
+const navlisteners2 = new Navlisteners(dataContainer, contactSection,
+  formSubmision, dataContainerTitle);
 
-  removeBook(removeButton, index) {
-    dataContainer.removeChild(removeButton.parentElement);
-    this.books.splice(index, 1);
-    localStorage.setItem('books', JSON.stringify(this.books));
-    location.reload();
-  }
-
-  static ListRemove() {
-    dataContainer.style.display = 'flex';
-    contactSection.style.display = 'none';
-    formSubmision.style.display = 'none';
-    dataContainerTitle.style.display = 'flex';
-
-    if (dataContainer.childElementCount === 0) {
-      dataContainer.style.display = 'none';
-    }
-  }
-
-  static addNewRemove() {
-    dataContainer.style.display = 'none';
-    formSubmision.style.display = 'flex';
-    dataContainerTitle.style.display = 'none';
-    contactSection.style.display = 'none';
-  }
-
-  static ContactRemove() {
-    formSubmision.style.display = 'none';
-    dataContainer.style.display = 'none';
-    contactSection.style.display = 'flex';
-    dataContainerTitle.style.display = 'none';
-  }
+function clock() {
+  const dt = DateTime.now();
+  time.innerHTML = dt.toLocaleString(DateTime.DATETIME_MED_WITH_SECONDS);
 }
```

Comment:
- [ ] Kindly refactor this to an arrow function in line with the ES6 syntax requirement.

Reply:
well resolved! ✔️